zoo/dmc2gym/envs/dmc2gym_lightzero_env.py

```diff
@@ -300,7 +322,28 @@ def reset(self) -> Dict[str, np.ndarray]:
                 self._env = FrameStackWrapper(self._env, self._cfg['frame_stack'])
 
             # set the obs, action space of wrapped env
-            self._observation_space = self._env.observation_space
+            self._observation_space = gym.spaces.Dict({
```

Comment:
why do we need to set this variable twice?

Reply:
This is a previous bug, we have fixed it now.